indy_client/test/cli/conftest.py

```diff
@@ -1349,19 +1301,19 @@ def philCli(be, do, philCLI, trusteeCli, poolTxnData):
 
 
 @pytest.fixture(scope="module")
-def faberAddedByPhil(be, do, poolNodesStarted, philCli, connectedToTest,
```

Comment:
Why did we remove `connectedToTest`?

Reply:
we do not use pool network named "test" anymore